python/lsst/afw/image/basicUtils.py

```diff
@@ -106,5 +112,70 @@ def assertWcsNearlyEqualOverBBox(testCase, wcs0, wcs1, bbox, maxDiffSky=0.01*afw
     if measDiffPix[0] > maxDiffPix:
         msgList.append("%s max measured pix error > %s max allowed pix error at sky pos=%s" %
                 (measDiffPix[0], maxDiffPix, measDiffPix[1]))
-    if msgList:
-        testCase.fail("%s: %s" % (msg, "; ".join(msgList)))
+
+    return "; ".join(msgList)
+
+def wcsNearlyEqualOverBBox(wcs0, wcs1, bbox, maxDiffSky=0.01*afwGeom.arcseconds,
+    maxDiffPix=0.01, nx=5, ny=5):
+    """!Return True if two WCS are nearly equal over a grid of pixel positions
+
+    @param[in] wcs0  WCS 0 (an lsst.afw.image.Wcs)
+    @param[in] wcs1  WCS 1 (an lsst.afw.image.Wcs)
+    @param[in] bbox  boundaries of pixel grid over which to compare the WCSs (an lsst.afw.geom.Box2I or Box2D)
+    @param[in] maxDiffSky  maximum separation between sky positions computed using Wcs.pixelToSky
+        (an lsst.afw.geom.Angle)
+    @param[in] maxDiffPix  maximum separation between pixel positions computed using Wcs.skyToPixel
+    @param[in] nx  number of points in x for the grid of pixel positions
+    @param[in] ny  number of points in y for the grid of pixel positions
+    @param[in] doShortCircuit  if True then stop at the first error, else test all values in the grid
+        and return information about the worst violations found
+
+    @return None if the WCS are nearly equal, else return a string describing the largest error measured
```

Comment:
Thank you for catching this. I removed the `@return` documentation because it duplicated the summary. I also switched added an explicit bool cast to the value returned by _compareWcsOverBBox in hopes of making the intent a bit more obvious.